src/main/cljc/jarl/core.cljc

```diff
@@ -83,9 +82,9 @@
                                  (plan info data input))
                              (abort (error-msg (fmt/sprintf "IR contains more than one plan; please specify entrypoint %s"
                                                             (pr-str (map first plans))))))))]
-            (println (json/write-str result))))))
+            (println (json/write-str-pretty result))))))
     (catch ExceptionInfo e
       (println (ex-message e))
-      #?(:clj ((System/exit 1))))))
```

Comment:
Oops. Should have added some CLI unit tests 😅 .

Reply:
Heh, or we could have some e2e/system level tests to complement I suppose.